06_Day_Loops/06_day_starter/scripts/class.js

```diff
@@ -0,0 +1,67 @@
+//Excercise 1
+
+//1.
+for (let i = 0; i <= 10; i++) {
+  console.log(i)
+};
+
+
+let i = 0;
+while (i <= 10){
+  console.log(i)
+  i++
+};
+
+
+let i = 0;
+do {
+  console.log(i)
+  i++
+} while (i <= 10);
+
+
+//2.
+for (let i = 10; i >= 0; i-- ){
+    console.log(i) 
+};
+
+
+let i = 10;
+while (i >= 0){
+  console.log(i)
+  i--
+};
+
+
+let i = 10;
+do {
+  console.log(i)
+  i--
+} while(i >= 0);
+
+
+//3. 
+
+//4.
+for (let i = 0; i <= 6; i++){
+  console.log('#')
+};
+
+//5. 
+for (let i = 0; i <= 10; i++){
+  console.log(i * i)
+};
+
+//6.
+for (let i = 0; i <= 10; i++){
+  console.log(i * i * i)
+};
+
+//7. 
```

Comment:
If you know the answer to #8 then you know the answer to this. You can actually do it by changing just one character from your answer below